tehom/downloads.py

```diff
@@ -536,32 +536,69 @@ def _interpolate_and_group_ais(
     """Interpolate the lat/lon of ships to the specified time.
 
     Interpolation rules:
-        A ship has observations near the specified time, before and
+        C1: A ship has observations near the specified time, before and
             after: linear interpolation
-        A ship has one observation very near the specified time, either
+        C2: A ship has one observation very near the specified time, either
             before or after, but not both: constant interpolation
-        A ship does not meet above criteria: do not create an
+        C3: A ship does not meet above criteria: do not create an
             interpolated record for this ship at this time
 
     Note:
         What counts as "near" and "very near" is subject to change and
         may be refactored out into an interpolation parameters object
 
+        MWM: "near" = entries before and after the given time,
+             "very near" = one entry within a neighborhood of the given time
+
```

Comment:
Not sure what the point of this is.  These aren't definitions, despite the equals sign.  These are cases when each term is used.